### DISCUSSION TITLE
Adding helicopters to the enemy

### DISCUSSION THREAD

**Discussion by deBearN** [Vdauphin/HeartsAndMinds#1340] [Question and answer]
is there a way to have enemy Helicopters patrolling around? i have set both vanilla and modded heli's in the enemy arsenal, but neither shows up in the map

**Comment by Vdauphin** [comment]
Hello,

Just add them to the array `btc_type_motorized` like explained here https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/mission.sqf#L614

Cheers
